app/models/article.rb

```diff
@@ -41,6 +41,8 @@ class Article < ApplicationRecord
            class_name: "Comment"
   has_many :profile_pins, as: :pinnable, inverse_of: :pinnable
   has_many :buffer_updates, dependent: :destroy
+  has_many :html_variant_successes, dependent: :nullify
+  has_many :html_variant_trials, dependent: :nullify
```

Comment:
As the relation is optional between those two, I thought it best to nullify the key. Let me know if they should be destroyed instead